### PR TITLE
Task tm/j 20 remove redux toolkit boilerplate clutter

### PR DESCRIPTION
**Pull request by TarasMakarchuk**
In this task, I made a mistake and incorrectly indicated the name of the commits.
I had to specify **"J-20: - remove redux-toolkit boilerplate clutter"** and **"J-20: + add the .idea folder to the .gitignore file"**.

### CODE REVIEW

File: frontend/src/components/app/app.tsx [hunk 1]
```diff
@@ -1,79 +1,29 @@
-import { useLocation } from 'hooks/hooks';
+import React from 'react';
```
Comment (ghost) [inline frontend/src/components/app/app.tsx]:
```suggestion
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
don't forget about commit style

File: frontend/src/components/app/app.tsx [hunk 2]
```diff
@@ -1,79 +1,29 @@
...
             </li>
           </ul>
-          <p>Current path: {pathname}</p>
         </div>
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
           <Switch>
-            <Route path={AppRoute.ROOT} component={Counter} exact />
             <Route path={AppRoute.SIGN_IN} component={SignIn} exact />
             <Route path={AppRoute.SIGN_UP} component={SignUp} exact />
```
Comment (yevhenprybytok) [inline frontend/src/components/app/app.tsx]: I think we need to make the App component absolutely clean, even remove those links. Later we will add real routes here

**Review by yevhenprybytok — CHANGES_REQUESTED** [review]

File: frontend/src/components/app/app.tsx (hunk 1 above)
Comment (TarasMakarchuk) [reply frontend/src/components/app/app.tsx]: Thanks, removed the import

File: frontend/src/components/app/app.tsx (hunk 2 above)
Comment (TarasMakarchuk) [reply frontend/src/components/app/app.tsx]: Thanks, I removed everything from the component app.tsx

**Review by yevhenprybytok — DISMISSED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**